pkg/client/options.go

```diff
@@ -24,10 +24,11 @@ import (
 type Option func(*options)
 
 type options struct {
-	UserAgent   string
-	RetryCount  uint
-	InsecureTLS bool
-	Logger      interface{}
+	UserAgent           string
+	RetryCount          uint
+	InsecureTLS         bool
+	Logger              interface{}
+	NoDisableKeepalives bool
```

Comment:
Should this be `EnableKeepalives`?

Reply:
The field we're un-setting is `DisableKeepalives`. `EnableKeepalives` isn't really correct because it's not enabling `Keepalives`. The double negative seems more intuitive to me.